### PR TITLE
Enable cAdvisor ProcessMetrics collecting

### LINKED ISSUE #84308: container_sockets, container_threads, container_max_threads metrics always return 0
**What happened**:

container_sockets, container_threads, container_max_threads metrics were added as part of Kubernetes 1.16 release. 

However, I only see 0 for these metrics.

**What you expected to happen**:

These metrics should give non-zero numbers.

**How to reproduce it (as minimally and precisely as possible)**:

New cluster using Kubernetes 1.16.2 + Prometheus 2.11.1.

Scraped container_sockets, container_threads, container_max_threads metrics always return 0

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.16.2
- Cloud provider or hardware configuration: Azure, cluster provisioned using aks-engine
- OS (e.g: `cat /etc/os-release`): Ubuntu
- Kernel (e.g. `uname -a`):
- Install tools: aks-engine
- Network plugin and version (if this is a network-related bug):
- Others:

/sig api-machinery
/sig node

### PR DESCRIPTION
**Pull request by zouyee**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Enable cadvisor ProcessMetrics

**Which issue(s) this PR fixes**:

Fixes #84308

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/kubelet/cadvisor/cadvisor_linux.go [hunk 1]
```diff
@@ -94,6 +94,7 @@ func New(imageFsInfoProvider ImageFsInfoProvider, rootPath string, cgroupRoots [
 		cadvisormetrics.NetworkUsageMetrics:     struct{}{},
 		cadvisormetrics.AcceleratorUsageMetrics: struct{}{},
 		cadvisormetrics.AppMetrics:              struct{}{},
+		cadvisormetrics.ProcessMetrics:          struct{}{},
```
Comment (mattjmcnaughton) [inline pkg/kubelet/cadvisor/cadvisor_linux.go]: Out of curiosity, do you know why these weren't included before? Did we just forget?

**Review by mattjmcnaughton — COMMENTED** [review]
Thanks for your pr here!

Is there a test that we can add/update to validate these metrics are being recorded? Or in other words, how do we know this change has the intended impact :)

**Comment by jiayingz** [comment]
@zouyee I thought these metrics are already collected with https://github.com/kubernetes/kubernetes/pull/79002

**Comment by dashpole** [comment]
/priority important-soon

Whoops...
#79002 apparently only enabled collection of process metrics for the json cAdvisor endpoints.  Thanks for the fix.  It would be nice to write e2e tests for this endpoint as well.

/lgtm
/approve

**Comment by dashpole** [comment]
/assign @dchen1107 
for kubelet approval

File: pkg/kubelet/cadvisor/cadvisor_linux.go [hunk 2]
```diff
@@ -94,6 +94,7 @@ func New(imageFsInfoProvider ImageFsInfoProvider, rootPath string, cgroupRoots [
 		cadvisormetrics.NetworkUsageMetrics:     struct{}{},
 		cadvisormetrics.AcceleratorUsageMetrics: struct{}{},
 		cadvisormetrics.AppMetrics:              struct{}{},
```
Comment (tedyu) [inline pkg/kubelet/cadvisor/cadvisor_linux.go]: What about PerCpuUsageMetrics ?
I wonder if it should be added as well.

**Pull request merged**